test/driver_test.rb

```diff
@@ -116,21 +141,60 @@
     end
 
     it "correctly calculates the average rating" do
-      trip2 = RideShare::Trip.new(
-        id: 8,
-        driver: @driver,
-        passenger_id: 3,
-        start_time: Time.new(2016, 8, 8),
-        end_time: Time.new(2016, 8, 9),
-        rating: 1
-      )
-      @driver.add_trip(trip2)
+      @driver.add_trip(trip_2)
+
+      expect(@driver.average_rating).must_be_close_to (5.0 + 1.0) / 2.0, 0.01
+    end
+
+    it "ignores in progress trip with no rating" do
+      @driver.add_trip(trip_2)
+      @driver.add_trip(in_progress_trip)
 
       expect(@driver.average_rating).must_be_close_to (5.0 + 1.0) / 2.0, 0.01
```

Comment:
Consider grabbing the values (5.0 + 1.0) / 2.0, 0.01 from the variables you created in before so that if you change the data it is changed here too.